src/Symfony/Component/PropertyAccess/PropertyAccessor.php

```diff
@@ -928,7 +929,9 @@ public static function createCache($namespace, $defaultLifetime, $version, Logge
         }
 
         $apcu = new ApcuAdapter($namespace, $defaultLifetime / 5, $version);
-        if (null !== $logger) {
+        if ('cli' === PHP_SAPI && !ini_get('apc.enable_cli')) {
```

Comment:
I suggest using a fully-qualified name for the constant, so that OPCache can exclude this branch at compile time for other SAPIs